Error: File 'ui_application.ui' is not valid

while executing 'C:\Users\victo\Documents\Code\ShotPredictionApp\venv\Lib\site-packages\PySide6\uic -g python ui_application.ui'
